tests/eko/test_ev_operator.py

```diff
@@ -41,6 +41,7 @@ def test_quad_ker(monkeypatch):
             as0=2,
             as_raw=1,
             nf=3,
+            p=False,
```

Comment:
Since the fix is repeated, you see the pattern. If we see the pattern, let's refactor this @felixhekhorn